site/src/loci/Table.jsx

```diff
@@ -53,7 +53,7 @@ const cols = [
     name: "Description",
   },
   {
-    key: "position_hg38",
+    key: "position_base0_hg38",
     name: "Position hg38",
     render: (cell) => (
```

Comment:
Can use any value from the row here.

```
render: (cell, row) => 
   <Link to={`some-site.org/${row.differentField}`}>{cell}</Link
```

Reply:
Thanks! I'm getting `row is not defined`. Is it called something else? Or do I need to refer to the locus?

Reply:
I'd need to know more to be able to help, e.g. what is your exact code, are you seeing that error in the CLI or in the browser, what fields are you trying to access, what is your desired output on the site (link `href` and text).

Reply:
Here's the code. I want the cell to show the base0 and the link to go to base1. Same as in the locus pages (it's working there)


```
{
    key: "position_base0_hg38",
    name: "Position hg38",
    render: (cell) => (
      <Link
        to={`https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg38&position=${row.position_base1_hg38}`}
      >
        {cell}
      </Link>
    ),
  },
  ```

Reply:
You're missing the `row` argument from the function:

```
  render: (cell, row) =>
```

Other than that the code is right. Every variable in the code should be either imported from somewhere and taken from a function argument. There won't be any globals, except for vars on the browser `window.` object, but I always prefix those explicitly with `window.` to note that it's a global.

Reply:
Perfect, thank works. Thanks so much!!